service/src/main/java/greencity/service/HabitAssignServiceImpl.java

```diff
@@ -871,11 +872,20 @@ public HabitAssignDto cancelHabitAssign(Long habitId, Long userId) {
      * {@inheritDoc}
      */
     public void deleteHabitAssign(Long habitId, Long userId) {
```

Comment:
Please rename this method according to the functionality it does
And change Java docs